src/Button.vue

```diff
@@ -239,30 +184,31 @@ export default {
   },
   data () {
     return {
-      toggled: !!this.value
+      toggled: !!this.value,
+      defaultColorChecked: '#75c791',
```

Comment:
Not sure i understand this bit, seems you have this defined in another file aswell. Also what was wrong with having constants?